### PR TITLE
[Makefile] Rename `make ps` -> `make list-jobs`

### PR DESCRIPTION
**Pull request by ayushkovskiy**
for a regular user, `ps` means nothing. Moreover, `make ps` makes no sense if it runs only `neuro ps`. When renamed, it makes more sense.

### CODE REVIEW

**Comment by AlekseySh** [comment]
probably we have to grep only jobs related to this project?
and better name is `make list-project-jobs`

**Comment by ayushkovskiy** [comment]
> and better name is make list-project-jobs

I think this name is too long, since everything in Template relates to the project, let's drop this word.

**Comment by ayushkovskiy** [comment]
@mariyadavydova @AlekseySh 

Added grep, please re-review

**Comment by ayushkovskiy** [comment]
note, `grep` may be not available on Windows

File: {{cookiecutter.project_slug}}/Makefile [hunk 1]
```diff
@@ -203,6 +203,6 @@ lint:  ### Run static code analysis locally
 
 ##### MISC #####
 
-.PHONY: ps
-ps:  ### List all running and pending jobs
-	$(NEURO) ps
+.PHONY: list-jobs
+list-jobs:  ### List all running and pending jobs submitted in current project
+	$(NEURO) ps | grep $(PROJECT_POSTFIX) || true
```
Comment (ayushkovskiy) [inline {{cookiecutter.project_slug}}/Makefile]: What if there's another job with the same name but from another project?..

**Comment by ayushkovskiy** [comment]
Closing as `| grep "-$(PROJECT_POSTFIX)"` might include a non-project job if its `neuro-ps` output contains `-$(PROJECT_POSTFIX)`. Let's keep it simple for now.

**Pull request closed without merging**